### ISSUE TITLE
Found invalid chain at least ~6 blocks longer than our best chain

### ISSUE THREAD

**Issue by qi-zhou** [bitcoin-sv/docker-sv#1]
Hi,

I'm try to run Bitcoin-sv with docker-compose, I found it cant return  balance from api, here are some logs from bsv node: 
```
nodes_bsv_1_1  | 2020-02-06 12:21:06 [bitcoin-worker0-N-CAsyncTaskPool] UpdateTip: new best=000000000000000001d0bdb914d26f06e5a2b8330e875cb7bcb463649ae8c62f height=620870 version=0x20000000 log2_work=87.956841 tx=350031163 date='2020-02-06 12:20:35' progress=1.000000 cache=35.3MiB(261126txo) warning='54 of last 100 blocks have unexpected version'
nodes_bsv_1_1  | 2020-02-06 12:21:06 [bitcoin-worker0-N-CAsyncTaskPool] CheckForkWarningConditions: Warning: Found invalid chain at least ~6 blocks longer than our best chain.
nodes_bsv_1_1  | Chain state database corruption likely.
nodes_bsv_1_1  | 2020-02-06 12:21:15 [bitcoin-opencon] connect() to [2601:641:480:6340::1]:8333 failed: Cannot assign requested address (99)
```

And docker-compose file content as follows:

```
version: "3"
services:
  nodes_bsv:
    image: bitcoinsv/bitcoin-sv:1.0.1
    command: bitcoind
    healthcheck:
      test: ["CMD", "/entrypoint.sh", "bitcoin-cli", "getinfo"]
    ports:
      - "127.0.0.1:8333:8333"
    volumes:
      - ./data:/data
```

I have no idea how to repair it, who can help me? Thanks in advance!

**Comment by BarneyChambers** [comment]
I am having the same issue, I saw this article about it https://www.coinness.com/news/550844

Not sure how to disable safe mode though

**Comment by Danconnolly** [comment]
This issue was fixed in version 1.0.2. 

Please use the main bitcoin-sv repository for issues (https://github.com/bitcoin-sv/bitcoin-sv).
